test/correctness/document-correctness.py

```diff
@@ -341,9 +341,8 @@ def _run_operation_(op1, op2):
 
         if indexes_first:
             for i in indexes:
-                # When we do enable make sure we don't enable for 50% cases. That way unique indexes will cloud everything else. Make it something like 5-10%.
-                # uniqueIndex = gen.global_prng.choice([True, False])
-                uniqueIndex = False
+                # 5% likelyhood to be unique, assuming a uniform distribution
+                uniqueIndex = (gen.global_prng.randint(1,20) == 1)
```

Comment:
As we discussed offline, its probably good idea to reduce the probability much much less, something like 1 out of 1000 times, even less probably. As our insert path wouldn't really consider unique indexes, it's very likely every test with a unique index would fail on the duplicate document error and stop there. It will dominate the tests, and we wouldn't really get much coverage for queries.